includes/features/onboarding/class-wc-admin-onboarding.php

```diff
@@ -26,6 +26,43 @@ public static function get_instance() {
 		}
 		return self::$instance;
 	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_components_settings', array( $this, 'component_settings' ), 20 ); // Run after WC_Admin_Loader.
+	}
+
+	/**
+	 * Add alert count to the component settings.
+	 *
+	 * @param array $settings Component settings.
+	 */
+	public function component_settings( $settings ) {
+		$redirect_url = esc_url_raw(
+			add_query_arg(
+				array(
+					'page' => 'wc-admin',
+				),
+				admin_url( 'admin.php' )
+			) . '#/?step=details'
+		);
+
+		$settings['jetpackConnectUrl'] = Jetpack::init()->build_connect_url( true, $redirect_url, 'woocommerce-setup-wizard' );
+
+		// Redirect to local calypso, if we are developing locally.
```

Comment:
This is fantastic!  I'm wondering if there's another constant or flag we can use here as a backup since not everyone developing in wc-admin will have or want to run a local environment of Calypso.

Reply:
I didn't see anything existing in Jetpack we could use (just the query parameter).

I've added a `'WOOCOMMERCE_CALYPSO_LOCAL` flag for now, since you are right -- the number of developers who would want this is probably just limited to our team. We could document this on our Field Guide page on the p2.